judge/create_cgroups.in

```diff
@@ -9,6 +9,14 @@
 JUDGEHOSTUSER=@DOMJUDGE_USER@
 CGROUPBASE="/sys/fs/cgroup"
 
+# We do not need to do any of this with cgroup v2.
+fs_type=$(awk '$2 == "/sys/fs/cgroup" {print $3}' /proc/mounts)
+if [ "$fs_type" = "cgroup2" ]; then
+    echo "+memory" >> /sys/fs/cgroup/cgroup.subtree_control
+    echo "+cpuset" >> /sys/fs/cgroup/cgroup.subtree_control
+    exit 0
```

Comment:
Is it possible for these writes to fail? If so, I think this script should not exit with code `0`. Adding `set -e` would do this automatically, but perhaps we should use an (adapted version of) `cgroup_error_and_usage`? Because most likely, when these writes fail, the kernel parameters are not set.